src/Entities/EnderCrystal.cpp

```diff
@@ -33,6 +34,10 @@ void cEnderCrystal::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 {
 	UNUSED(a_Dt);
 	// No further processing (physics e.t.c.) is needed
+	if (m_World->GetDimension() == dimEnd)
+	{
+		m_World->SetBlock({POSX_TOINT, POSY_TOINT + 1, POSZ_TOINT}, E_BLOCK_FIRE, 0);
```

Comment:
Should we check the block already there == fire? Not sure if it's possible for pistons to push stuff into POSY_TOINT + 1 or to spawn a crystal where +1 is occupied? Otherwise it would be possible for crystals to act like an arbitrary block deleter.

Also POSITION_TOINT.addedY(1) should do it.

Reply:
The wiki states that any block is replaces with fire

https://minecraft.gamepedia.com/End_Crystal?mobileaction=toggle_view_desktop#Properties

Reply:
interesting, thanks.